src/Plugin/TwigPlugin/JumpstartUITwig.php

```diff
@@ -2,36 +2,89 @@
 
 namespace Drupal\jumpstart_ui\Plugin\TwigPlugin;
 
+use Drupal\Component\Render\MarkupInterface;
 use Drupal\Component\Utility\Html;
+use Drupal\Core\Render\RendererInterface;
 
 /**
  * Extend Drupal's Twig_Extension class.
  */
 class JumpstartUITwig extends \Twig_Extension {
 
+  /**
+   * Renderer service.
+   *
+   * @var \Drupal\Core\Render\RendererInterface
+   */
+  protected $renderer;
+
+  /**
+   * JumpstartUITwig constructor.
+   *
+   * @param \Drupal\Core\Render\RendererInterface $renderer
+   *   Renderer service.
+   */
+  public function __construct(RendererInterface $renderer = NULL) {
+    $this->renderer = $renderer;
+  }
+
   /**
    * {@inheritdoc}
    */
-  public function getFunctions() {
+  public function getFunctions(): array {
     return [
       new \Twig_SimpleFunction('getUniqueId', [$this, 'getUniqueId']),
     ];
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  public function getFilters(): array {
+    return [
+      new \Twig_SimpleFilter('render_clean', [$this, 'renderClean']),
+    ];
+  }
+
   /**
    * Generate a unique ID that won't be duplicated during this page render.
    *
-   * @param string $id
+   * @param string|null $id
    *   A CSS valid id string.
    *
    * @return string
    *   An Id that is unique to this page load.
    */
-  public function getUniqueId($id = NULL) {
+  public function getUniqueId(?string $id = NULL): string {
     if (is_null($id)) {
       $id = uniqid('jumpstart-ui-');
     }
     return Html::getUniqueId($id);
   }
 
+  /**
+   * Render the elements and strip all tags except those passed in.
+   *
+   * @param mixed $elements
+   *   Should be a render array.
+   * @param string $tags
+   *   Optionally which tags to keep.
+   *
+   * @return array|null
+   *   Markup render array.
+   */
+  public function renderClean($elements, $tags = '<drupal-render-placeholder>') {
+    if ($elements instanceof MarkupInterface) {
+      $elements = ['#markup' => (string) $elements];
+    }
+
+    $rendered = $this->renderer->render($elements);
+    if (strpos($tags, '<drupal-render-placeholder>') === FALSE) {
+      $tags .= '<drupal-render-placeholder>';
+    }
```

Comment:
🤔 Is there a reason why we would want to strip everything including the placeholder?

Reply:
The question I am getting at is do we want to add the placeholder at all times or should we be more explicit. I usually lean towards explicit.

Reply:
Meh. I dont see much either way. Honestly I still haven't seen that tag. I've just trusted some documentation